packages/core/docs/advanced/server-middleware.md

```diff
@@ -75,6 +75,7 @@ const extension = {
   extendApiMethods: {
     getProduct: async () => { /* ... */ }
   },
+  extendApp: (app) => {  /* ... */ },
```

Comment:
Why extension can extend the app but integration cannot? I imagine we will end up having many 1+1 integrations (1 integration with 1 extension - because we want to extend express' app)

Reply:
_but integration cannot?_ - yes they can, extensions are defined in the middleware config.